src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -88,8 +96,9 @@ export default class ZipPaymentStrategy implements PaymentStrategy {
             } catch (error) {
                 if (error instanceof RequestError && error.body.status === 'additional_action_required') {
                     return new Promise(() => {
```

Comment:
You can get rid of this as well

Reply:
♻️  @animesh1987 thanks for your comments, I've changed this with your suggestions.